sample_configs/embedding_advanced.yml

```diff
@@ -0,0 +1,6 @@
+language: en
+pipeline:
```

Comment:
why is this one "advanced" 😆 ?

Reply:
I believe it is the test config,  @amn41 ?

Reply:
oh because it specifies the components one-by-one rather than the predefined pipeline

Reply:
I moved it to data/test folder since it is only used in the test